lib/purchases_flutter.dart

```diff
@@ -342,8 +356,9 @@ class Purchases {
   ///
   /// [packageToPurchase] The Package you wish to purchase
   ///
-  /// [upgradeInfo] Android only. Optional UpgradeInfo you wish to upgrade from
-  /// containing the oldSKU and the optional prorationMode.
+  /// [productChangeInfo] Android only. Optional ProductChangeInfo you wish to
+  /// change from containing the oldProductIdentifer and the
+  /// optional prorationMode.
```

Comment:
this will default to IMMEDIATE_WITHOUT_PRORATION at the native level, we should call that out.